WcaOnRails/app/controllers/posts_controller.rb

```diff
@@ -5,7 +5,13 @@ class PostsController < ApplicationController
   before_action -> { redirect_to_root_unless_user(:can_create_posts?) }, except: [:index, :rss, :show]
 
   def index
-    @posts = Post.where(world_readable: true).order(sticky: :desc, created_at: :desc).includes(:author).page(params[:page])
+    tag = params[:tag]
+    if tag
+      @posts = Post.joins(:post_tags).where('post_tags.tag = ?', tag)
+    else
+      @posts = Post.joins("LEFT JOIN post_tags ON post_tags.post_id = posts.id").group("posts.id").having("IFNULL(SUM(tag = 'wdc'), 0) = 0")
+    end
+    @posts = @posts.where(world_readable: true).order(sticky: :desc, created_at: :desc).includes(:author).page(params[:page])
```

Comment:
By "the others", are you referring to "posts that have `world_readable == false`"? We have only one post where world_readable is false, it's the [delegate crash course](https://github.com/thewca/worldcubeassociation.org/blob/4fca964054af61ebb4e7dd8b0fef6b3dd7a1b0bf/WcaOnRails/app/models/post.rb#L32). I'm not in love with the complexity added by the delegate crash course, but I don't think we should be trying to improve that in this PR either =) If you have ideas for making it better, we can continue to chat here, or feel free to make a new issue to talk about it. (it's also possible I'm misunderstanding what you were asking about here)